<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgPlaylists</class>
 <widget class="QDialog" name="DlgPlaylists">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Плейлисты</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="cbPList">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnNewPlaylist">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Н&amp;овый...</string>
       </property>
       <property name="icon">
        <iconset resource="AdManager.qrc">
         <normaloff>:/icons/New.png</normaloff>:/icons/New.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCopyPlaylist">
       <property name="text">
        <string>Копи&amp;я</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRenPlaylist">
       <property name="text">
        <string>Пере&amp;именовать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDelPlaylist">
       <property name="text">
        <string>&amp;Удалить</string>
       </property>
       <property name="icon">
        <iconset resource="AdManager.qrc">
         <normaloff>:/icons/Delete.png</normaloff>:/icons/Delete.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabContent">
      <attribute name="title">
       <string>&amp;Контент</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QListWidget" name="plContent">
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="btnAddItems">
           <property name="text">
            <string>&amp;Добавить...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDelItems">
           <property name="text">
            <string>Удал&amp;ить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnItemProps">
           <property name="text">
            <string>&amp;Свойства...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnMoveUp">
           <property name="text">
            <string>Вверх</string>
           </property>
           <property name="icon">
            <iconset resource="AdManager.qrc">
             <normaloff>:/icons/Up.png</normaloff>:/icons/Up.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnMoveDown">
           <property name="text">
            <string>Вниз</string>
           </property>
           <property name="icon">
            <iconset resource="AdManager.qrc">
             <normaloff>:/icons/Down.png</normaloff>:/icons/Down.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSettings">
      <attribute name="title">
       <string>&amp;Настройки</string>
      </attribute>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>241</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Случайный порядок воспроизведения</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>226</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Вермя демонстрации для изображений</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>170</y>
         <width>81</width>
         <height>24</height>
        </rect>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::PlusMinus</enum>
       </property>
       <property name="suffix">
        <string> сек</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>3600</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>279</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Удалять неиспользуемые файлы с устройства</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbPList</tabstop>
  <tabstop>btnNewPlaylist</tabstop>
  <tabstop>btnCopyPlaylist</tabstop>
  <tabstop>btnRenPlaylist</tabstop>
  <tabstop>btnDelPlaylist</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>plContent</tabstop>
  <tabstop>btnAddItems</tabstop>
  <tabstop>btnDelItems</tabstop>
  <tabstop>btnItemProps</tabstop>
  <tabstop>btnMoveUp</tabstop>
  <tabstop>btnMoveDown</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>checkBox_2</tabstop>
  <tabstop>spinBox</tabstop>
 </tabstops>
 <resources>
  <include location="AdManager.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DlgPlaylists</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgPlaylists</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
